EditorConfigGenerator.Core/Styles/CSharpSpaceAfterKeywordsInControlFlowStatementsStyle.cs

```diff
@@ -39,7 +39,9 @@ public override CSharpSpaceAfterKeywordsInControlFlowStatementsStyle Update(Toke
 
 			var token = information.Token;
 
-			if(!token.Parent.ContainsDiagnostics)
+#pragma warning disable CS8602 // Dereference of a possibly null reference.
+			if (!token.Parent.ContainsDiagnostics)
+#pragma warning restore CS8602 // Dereference of a possibly null reference.
```

Comment:
I'll accept this in the PR, but I'd rather not ignore NRT-related warnings. I'll make the changes appropriately in this and other files.